utilities/mdbook-gen-summary/mdbook-gen-summary.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+# Simple program to create an index for rust doc documentation, to be embedded inside an md-book
+# We do not use any non-standard packages, so this script should "just work" if
```

Comment:
trailing "if", is there something that's meant to be here?

Reply:
fixed